### PR TITLE
Clientmanagement

### PR DESCRIPTION
**Pull request by wcampbell09**

### CODE REVIEW

File: modules/clientmanagements/client/config/clientmanagements.client.routes.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      .state('clientmanagements.create', {
+        url: '/create',
+        templateUrl: 'modules/clientmanagements/client/views/form-clientmanagement.client.view.html',
+        controller: 'ClientmanagementsController',
+        controllerAs: 'vm',
+        resolve: {
+          clientmanagementResolve: newClientmanagement
+        },
+        data: {
+          roles: ['user', 'admin'],
```
Comment (angryoctopode) [inline modules/clientmanagements/client/config/clientmanagements.client.routes.js]: You can just throw this under the abstract state, 'clientmanagements'

File: modules/clientmanagements/client/config/clientmanagements.client.routes.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      .state('clientmanagements.edit', {
+        url: '/:clientmanagementId/edit',
+        templateUrl: 'modules/clientmanagements/client/views/form-clientmanagement.client.view.html',
+        controller: 'ClientmanagementsController',
+        controllerAs: 'vm',
+        resolve: {
+          clientmanagementResolve: getClientmanagement
+        },
+        data: {
+          roles: ['user', 'admin'],
```
Comment (angryoctopode) [inline modules/clientmanagements/client/config/clientmanagements.client.routes.js]: You can just throw this under the abstract state, 'clientmanagements'

File: modules/clientmanagements/client/config/clientmanagements.client.routes.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+  routeConfig.$inject = ['$stateProvider'];
+
+  function routeConfig($stateProvider) {
+    $stateProvider
+      .state('clientmanagements', {
+        abstract: true,
+        url: '/clientmanagements',
+        template: '<ui-view/>'
+      })
```
Comment (angryoctopode) [inline modules/clientmanagements/client/config/clientmanagements.client.routes.js]: add data: {
  roles: [/* Insert roles here */]
}

File: modules/clientmanagements/client/views/list-clientmanagements.client.view.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+                <tr>
+                    <th>Name</th>
+                    <th>Phone</th>
+                    <th>Email</th>
+                    <th>Dog Name</th>
+                    <th>Dog ID</a></th>
+                </tr>
+              </thead>
+                <tbody>
+                  <tr ng-repeat="clientmanagement in vm.clientmanagements | filter: query" ui-sref= "clientmanagements.view({clientmanagementId:
```
Comment (dkwhite200) [inline modules/clientmanagements/client/views/list-clientmanagements.client.view.html:29]: Fix this indentation

File: modules/clientmanagements/client/config/clientmanagements.client.routes.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    .config(routeConfig);
+
+  routeConfig.$inject = ['$stateProvider'];
+
+  function routeConfig($stateProvider) {
+    $stateProvider
+      .state('clientmanagements', {
+        abstract: true,
+        url: '/clientmanagements',
+        template: '<ui-view/>'
```
Comment (angryoctopode) [inline modules/clientmanagements/client/config/clientmanagements.client.routes.js]: add data: {
roles: [/* Insert roles here */]
}

**Review by eristow — APPROVED** [review]
All is good.

File: modules/clientmanagements/client/config/clientmanagements.client.routes.js (hunk 5 above)
Comment (AriTheGuitarMan) [reply modules/clientmanagements/client/config/clientmanagements.client.routes.js]: Logical

**Review by dkwhite200 — APPROVED** [review]
Go for it

**Review by wcampbell09 — COMMENTED** [review]
Good stuff

**Review by angryoctopode — CHANGES_REQUESTED** [review]
Please fix these app breaking things, ASAP.

File: modules/clientmanagements/client/config/clientmanagements.client.routes.js (hunk 5 above)
Comment (angryoctopode) [reply modules/clientmanagements/client/config/clientmanagements.client.routes.js]: Looks gucci now. 👍

Comment (angryoctopode) [reply modules/clientmanagements/client/config/clientmanagements.client.routes.js]: 0/10. Missing comma after line. 👎

Comment (dkwhite200) [reply modules/clientmanagements/client/config/clientmanagements.client.routes.js]: MY EYES

**Review by dkwhite200 — APPROVED** [review]
Nice Job

**Review by angryoctopode — APPROVED** [review]

**Review by AriTheGuitarMan — APPROVED** [review]
The table looks nice. Good. Make sure routes work well.

**Pull request merged**